src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -65,6 +65,25 @@ public void storeLocation(Location location) {
         datastore.put(locationEntity);
     }
 
+
```

Comment:
Please consider removing this extra line.

Reply:
Just to clarify: when I am strict about extra lines, it's when you added or removed a line in an unrelated part of the code. Here, it's related - right below, you're adding a getLocations method.

Having said that, it's not necessary to have 2 lines in between methods, so feel free to remove this.